core/src/test/java/org/apache/accumulo/core/metadata/schema/TabletMetadataTest.java

```diff
@@ -102,7 +103,7 @@ public void testAllColumns() {
     assertEquals(Map.of("df1", dfv1, "df2", dfv2), tm.getFilesMap());
     assertEquals(6L, tm.getFlushId().getAsLong());
     assertEquals(rowMap, tm.getKeyValues());
-    assertEquals(Set.of("bf1", "bf2"), Set.copyOf(tm.getLoaded().keySet()));
+    assertEquals(Map.of("bf1", 56L, "bf2", 59L), tm.getLoaded());
```

Comment:
Does ordering matter for this check? One thing I've seen is that `Map.of` does not necessarily preserve "insertion order" (order based on args).

Reply:
I would not expect `Map.equals()` to care about the insertion order.